src/main/java/com/google/firebase/remoteconfig/Version.java

```diff
@@ -69,19 +66,16 @@ private Version() {
     if (!Strings.isNullOrEmpty(versionResponse.getUpdateTime())) {
       // Update Time is a timestamp in RFC3339 UTC "Zulu" format, accurate to nanoseconds.
       // example: "2014-10-02T15:01:23.045123456Z"
-      // SimpleDateFormat cannot handle nanoseconds, therefore we drop nanoseconds from the string.
-      Matcher errorMatcher = ZULU_TIME_NO_NANOSECONDS_PATTERN
-              .matcher(versionResponse.getUpdateTime());
-      String updateTimeWithoutNanoseconds = "";
-      if (errorMatcher.find()) {
-        updateTimeWithoutNanoseconds = errorMatcher.group(1);
-      }
+      // SimpleDateFormat cannot handle nanoseconds, therefore we strip nanoseconds from the string.
+      String updateTime = versionResponse.getUpdateTime();
+      int indexOfPeriod = !updateTime.contains(".") ? 0 : updateTime.indexOf(".");
+      String updateTimeWithoutNanoseconds = updateTime.substring(0, indexOfPeriod);
```

Comment:
indexOfPeriod can be possibly 0, in which case this will do `substring(0, 0)`.  I think following makes more sense to me:

```java
String updateTime = versionResponse.getUpdateTime();
int indexOfPeriod = updateTime.indexOf('.');
if (indexOfPeriod != -1) {
  updateTime = updateTime.substring(0, indexOfPeriod);
}
```

Reply:
I agree! This is better. Update the code. Thanks!